src/app/space/app-launcher/services/app-launcher-project-summary.service.ts

```diff
@@ -43,23 +40,15 @@ export class AppLauncherProjectSummaryService implements ProjectSummaryService {
   /**
    * Set up the project for the given summary
    *
-   * @param  {Summary} summary
-   * @param  {string} spaceId
-   * @param  {string} spaceName
-   * @param  {boolean} isImport
-   * @returns Observable
+   * @param {Projectile} projectile The project summary
+   * @returns {Observable<boolean>}
    */
-  setup(summary: Summary, retry?: number): Observable<any> {
+  setup(projectile: Projectile<any>, retry?: number): Observable<any> {
```

Comment:
Why does this return `Observable<any>`? Do we not have a concrete type available?

Reply:
We don't have any concrete type available for Projectile in ```ngx-launcher```

Reply:
I was asking about the return type.

Reply:
I looked at the call to `setup` in `ngx-launcher` and it seems to expect an any type but then proceeds to use values from the object.
Your code is fine because the api aligns with `ngx-launcher`.